pages/courses_page.py

```diff
@@ -38,3 +36,24 @@ def enroll_course(self, name, full_course_name):
     def verify_enroll_successful(self, full_course_name):
         self.navigate_to_page(DASHBOARD)
         return self.is_element_present(_my_course.format(full_course_name))
+
+    def verify_upgrade_enrollment(self, full_course_name):
+        self.element_click(_my_course.format(full_course_name))
+        self.wait_for_element(upgrade_link)
+        self.element_click(upgrade_link)
+        self.wait_for_element(first_name)
+        self.enter_random_data(first_name, 7)
+        self.enter_random_data(last_name, 7)
+        self.enter_random_data(address, 7)
+        self.enter_random_data(unit, 7)
+        self.enter_random_data(city, 7)
+        self.select_value_from_dropdown(country, 'US')
+        self.select_value_from_dropdown(state, 'MA')
+        self.enter_random_data(postal_code, 5, type='digits')
+        self.enter_random_data(card_number, 20, type='digits')
+        self.enter_random_data(security_code, 3, type='digits')
```

Comment:
Please change all type to data_type. There is no 'type' argument in enter_random_data